docs/team_strategy/facilitator_notes.md

```diff
@@ -0,0 +1,69 @@
+# Facilitator notes
+
+These notes are a comprehensive guide on how to use all our team strategy templates *as presented here*. 
+They contain both information on how to run the sessions as presented operationally (Facilitator notes), as well as insights and lessons learned from the RAM team and contributors when using them (Insights).
+
+> :warning: **NOTE**: These templates are being dynamically built through direct usage across project teams and contexts. They are **not** a definitive source of truth on how to run successful strategy sessions for your team. They are more intended as starting points for you to amend/use as required. Any improvements or changes you make, please contribute them upstream back to this repository, so we can spread the love on effective team strategising!
+
+These notes are a deeper dive than other resources we have. For an overview of our approach, see [this page](./index.md) or our [subchapter in the Turing Way](https://the-turing-way.netlify.app/collaboration/stakeholder-engagement.html#strategy-alignment-workshops).
+
+## Before any sessions
+
+### :check_mark_button: Facilitator notes
+There are a number of things to make sure you have in place before undertaking any sessions.
+* **Schedule for all sessions**: You should know how many sessions you plan to run, when you plan to run them, and schedule these all in advance. This guide is designed for 4 x 2 hours sessions, which we recommend scheduling over approximately 1/1.5 months. However this will need to flex depending on your stakeholders.
+* **Stakeholder engagement and review**: The most important part of pre-session work. Ensure you understand the needs, wants and aims of your stakeholders, as well as their approach to this work. If they are fully bought in and have time, you can likely run more, in depth sessions. If they are busy/not fully bought in, you might need to pick on or two key sessions to run. You should also get feedback on your proposed strategy plans ahead of any session, to ensure co-creation of team strategy from the very beginning.
```

Comment:
I think I'd prefer using "Participant" in these resources instead of "Stakeholder". What do you think?

Reply:
Yessss get me away from stakeholder!!